models/token.go

```diff
@@ -21,11 +21,11 @@ type AccessToken struct {
 	Sha1 string `xorm:"UNIQUE VARCHAR(40)"`
 
 	Created           time.Time `xorm:"-"`
-	CreatedUnix       int64
+	CreatedUnix       int64     `xorm:"INDEX"`
```

Comment:
If my understanding of database indices is correct, a non-clustered index will not help `ORDER BY` clauses.